components/collator/src/provider.rs

```diff
@@ -537,18 +556,68 @@ pub struct CollationSpecialPrimaries<'data> {
     /// character classes packed so that each fits in
     /// 16 bits. Length must match the number of enum
     /// variants in `MaxVariable`, currently 4.
+    ///
+    /// This is potentially followed by 256 bits
```

Comment:
It would probably make sense to say that "potentially" is only about the case of using `icu_collator_data` 2.0.0 and in all other cases, it's a data generation bug for the extra data not to be there.